src/apiGatewayTest/java/uk/gov/hmcts/reform/bulkscanprocessor/api/GetSasTokenTest.java

```diff
@@ -59,6 +59,7 @@ public void should_reject_request_lacking_subscription_key() throws Exception {
         assertThat(response.body().asString()).contains("Access denied due to missing subscription key");
     }
 
+    @Test
```

Comment:
ahh this was missing so it will be nice to see if the test works consistently now 😉